### DISCUSSION TITLE
SecondPhase

### DISCUSSION THREAD

**Discussion by Adairle** [idaholab/moose#24591] [Q&A Modules: Phase field] [answered]
![image](https://github.com/idaholab/moose/assets/115397737/d46441c1-e05c-4320-8f5d-9dcd7dbb9252)
![image](https://github.com/idaholab/moose/assets/115397737/05922444-1746-4ce2-95ca-bd6791bbc42e)
Request help
The collision program will end after the interaction force between the second phase particles at the solid-liquid interface.
Looking forward to your reply, thank you！

**Comment by GiudGiud** [comment, marked as answer]
Hello

Could you please give more details? What models are you using?

Guillaume
